plugins/kotlindoc-gradle-plugin/src/main/groovy/org/kordamp/gradle/plugin/kotlindoc/KotlindocPlugin.groovy

```diff
@@ -262,6 +262,7 @@ class KotlindocPlugin extends AbstractKordampPlugin {
                         t.dokkaRuntime = project.configurations.findByName(DOKKA_RUNTIME_CONFIGURATION_NAME)
                         t.extensions.add('multiplatform', project.container(GradlePassConfigurationImpl))
                         t.extensions.add('configuration', new GradlePassConfigurationImpl())
+                        t.subProjects = project.subprojects.collect { it.name }.toList()
```

Comment:
I think this should be all subprojects that have the `org.jetbrains.kotlin.jvm` plugin applied.